src/jit/compiler.cpp

```diff
@@ -2403,31 +2403,32 @@ unsigned ReinterpretHexAsDecimal(unsigned in)
     return result;
 }
 
-void Compiler::compInitOptions(CORJIT_FLAGS* jitFlags)
+void Compiler::compInitOptions(JitFlags* jitFlags)
 {
 #ifdef UNIX_AMD64_ABI
     opts.compNeedToAlignFrame = false;
 #endif // UNIX_AMD64_ABI
     memset(&opts, 0, sizeof(opts));
 
-    unsigned compileFlags = jitFlags->corJitFlags;
-
     if (compIsForInlining())
     {
-        assert((compileFlags & CORJIT_FLG_LOST_WHEN_INLINING) == 0);
-        assert(compileFlags & CORJIT_FLG_SKIP_VERIFICATION);
+        assert(!jitFlags->IsSet(JitFlags::JIT_FLAG_BBOPT));
```

Comment:
These are now broken out separately here and in flowgraph.cpp. Would it make sense to add methods to `JitFlags` that set/clear and assert the appropriate flags for inlining? That way they would be in close proximity and more easily kept in sync.